picture-stealer/src/main/java/com/bobocode/picturestealer/manager/PhotoRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.bobocode.picturestealer.manager;
+
+import com.bobocode.picturestealer.entity.PhotoEntity;
+import jakarta.persistence.QueryHint;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.jpa.repository.QueryHints;
+
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * @author "Maksym Oliinyk"
+ */
+public interface PhotoRepository
+        extends JpaRepository<PhotoEntity, UUID> {
+
+    @Query("select p from PhotoEntity p where p.nasaId = :nasaId")
```

Comment:
I'm confused. Why do you need a custom query? I assumed if you remove it it will continue to work or I missed something.

Reply:
Yes, it will work, you can check CameraRepository